build/Targets/Settings.props

```diff
@@ -169,6 +163,19 @@
     <CustomAfterMicrosoftCommonTargets>$(MSBuildThisFileDirectory)\AfterCommonTargets.targets</CustomAfterMicrosoftCommonTargets>
   </PropertyGroup>
 
+  <!-- Determine compiler toolset props file to use for this build -->
+  <PropertyGroup>
+    <_PackageName Condition="'$(MSBuildRuntimeType)' != 'Core'">Microsoft.Net.Compilers</_PackageName>
+    <_PackageName Condition="'$(MSBuildRuntimeType)' == 'Core'">Microsoft.NETCore.Compilers</_PackageName>
+    <_PackageVersion Condition="'$(MSBuildRuntimeType)' != 'Core' AND '$(BootstrapBuildPath)' == ''">$(MicrosoftNetCompilersVersion)</_PackageVersion>
+    <_PackageVersion Condition="'$(MSBuildRuntimeType)' == 'Core' AND '$(BootstrapBuildPath)' == ''">$(MicrosoftNETCoreCompilersVersion)</_PackageVersion>
+    <_PackageVersion Condition="'$(BootstrapBuildPath)' != ''">1.0.0</_PackageVersion>
+    <_PackageRoot>$(NuGetPackageRoot)</_PackageRoot>
+    <_PackageRoot Condition="'$(BootstrapBuildPath)' != ''">$(BootstrapBuildPath)</_PackageRoot>
+    <RoslynToolsetPropsDirectory>$(_PackageRoot)/$(_PackageName.ToLower())/$(_PackageVersion)/build</RoslynToolsetPropsDirectory>
```

Comment:
Should we save `$(_PackageRoot)/$(_PackageName.ToLower())/$(_PackageVersion)` instead, and add either `build` or `tools` later?
Currently, we add `build` here, but later remove it to add `..\tools` instead (line 202).